.htaccess

```diff
@@ -30,12 +30,11 @@ DirectoryIndex index.php index.html index.htm
 
 # PHP 5, Apache 1 and 2.
 <IfModule mod_php5.c>
-  php_flag magic_quotes_gpc                 off
   php_flag magic_quotes_sybase              off
-  php_flag register_globals                 off
   php_flag session.auto_start               off
   php_value mbstring.http_input             pass
   php_value mbstring.http_output            pass
+  php_value max_input_vars                  2003
```

Comment:
I'd go with something larger like 10000. That's the value I use on most of my sites to be safe. Pantheon defaults to that as well.